### PR TITLE
Add Red Bull Presence

### LINKED ISSUE #4206: Red Bull TV | www.redbull.com
**Prerequisites and essential questions**
- [x] Is it a popular site?
- [x] Is the website older than 2 months?
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?**
Krypton#7331 (ID: 562359123086409729)

**What's the name of the service?**
www.redbull.com | Red Bull TV

**What should the Presence display?**
The title of the current event or short film you're watching and eventually the time elapsed and the total length of the video.
When browsing a category it could display "Browsing category {category}"

**If possible, please provide a logo for the service (512x512 minimum)**
Red-Bull-TV

Banner:
Red-Bull-TV

### PR DESCRIPTION
**Pull request by vahya**
Screenshot 2021-08-05 131159
Screenshot 2021-08-05 131231
Screenshot 2021-08-05 131256
Screenshot 2021-08-05 131325
Screenshot 2021-08-05 131351
Screenshot 2021-08-05 131432

Closes #4206

### CODE REVIEW

**Comment by kkrypt0nn** [comment]
Thanks!
Looked at the code and have seen that you have "Looking at article {article}" when you're _reading_ an article, would be nice to have "Reading article {article}" instead :)

**Review by Bas950 — CHANGES_REQUESTED** [review]
Support all pages not just red bull tv? Since its all on the same domain as the drink etc. (no <tv>.redbull.com).

**Comment by vahya** [comment]
> Support all pages not just red bull tv? Since its all on the same domain as the drink etc. (no .redbull.com).

All the pages are supported 😅

**Comment by Bas950** [comment]
> > Support all pages not just red bull tv? Since its all on the same domain as the drink etc. (no .redbull.com).
> 
> All the pages are supported 😅

https://www.redbull.com/nl-nl/energydrink
https://www.redbull.com/nl-nl/energydrink/red-bull-red-edition
https://www.redbull.com/nl-nl/energydrink/red-bull-red-edition-ingredienten

dont think these pages are supported, and rename it to just Redbull, like I said support the whole redbull website, not just the TV section.

File: websites/R/Red Bull/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "Red Bull",
+  "author": {
+    "name": "वाह्य",
+    "id": "819696394180100106"
+  },
+  "category": "videos",
+  "logo": "https://i.imgur.com/sAxySI0.png",
```
Comment (Bas950) [inline websites/R/Red Bull/dist/metadata.json:9]: update logo so it isnt only for redbulltv

Comment (Bas950) [reply websites/R/Red Bull/dist/metadata.json:9]: Same with banner and presence assets

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**